autopush/settings.py

```diff
@@ -322,9 +324,15 @@ def parse_endpoint(self, token, version="v0", ckey_header=None):
         public_key = None
         if ckey_header:
             try:
-                public_key = CryptoKey(ckey_header).get_label('p256ecdsa')
+                crypto_key = CryptoKey(ckey_header)
             except CryptoKeyException:
                 raise InvalidTokenException("Invalid key data")
+            label = crypto_key.get_label('p256ecdsa')
+            try:
+                public_key = base64url_decode(label)
+            except:
```

Comment:
What exception can be thrown here? Generally bare excepts are to be avoided when possible.